dipy/workflows/segment.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import division, print_function, absolute_import
+
+from glob import glob
+from os.path import join, basename, splitext, dirname, isabs, exists
+from os import makedirs
+
+import nibabel as nib
+import numpy as np
+
+from dipy.segment.mask import median_otsu
+
+
+def median_otsu_bet(input_file, out_dir, save_masked=False, median_radius=4,
+                    numpass=4, autocrop=False, vol_idx=None, dilate=None):
+
+    for fpath in glob(input_file):
```

Comment:
Because this allows it to run with multiple files it would be nice to have some print out too. Showing what is being currently processed.